server/src/instant/fixtures.clj

```diff
@@ -26,14 +28,37 @@
       :params {:app_id (:id a)}
       :body {}})))
 
+(defmacro with-indexing-job-queue [job-queue & body]
```

Comment:
Moved this into fixtures and use it for every `with-x-app` so that we clean up the queue before moving on to the next test.